lib/pokerscore/hand.rb

```diff
@@ -35,20 +35,24 @@ def invalid_hand?(cards)
     unique_cards.length != 5
   end
 
+  def flush?
+    flush = true
```

Comment:
Could you think of a way to use a combination of `group_by/map` and `uniq` on the cards to make this more succinct?